CODE_STYLE.md

```diff
@@ -2,3 +2,14 @@
 
 * [Ruby](https://github.com/bbatsov/ruby-style-guide)
 * [Objective-C](https://github.com/hyperoslo/objective-c-style-guide)
+* [JavaScript](https://github.com/hyperoslo/eslint-config)
+
+## Stylus
+
+We have not yet chosen a style guide for Stylus, other than to decide that:
+
+* We *don't* use `;`.
+* We *do* use `{` and `}`.
```

Comment:
:+1: on landing something that fits everybody here. The reasoning behind rolling with `{`s on some projects has been to lower mental overhead when swapping between strict ES6 and #do-whatever languages like Stylus. Personally fine with whatever we end up with 👍